publisher-ui.md

```diff
@@ -0,0 +1,51 @@
+`create_settings_widget(parent)`
```

Comment:
This doc should be added to the [plugin interface documentation](https://support.shotgunsoftware.com/hc/en-us/articles/115000067513-Integrations-Developer-Guide#Publish%20Plugins).